layouts/partials/sections/recent-posts.html

```diff
@@ -24,4 +24,13 @@ <h1 class="text-center" style="display: none">
       {{ end }}
     </div>
   </div>
+  {{ if (.section.showButton) }}
```

Comment:
Should we rename `showButton` to `showMoreButton`?

Reply:
Yes, this name is more representative.